app/decorators/application_decorator.rb

```diff
@@ -1,10 +1,49 @@
 # frozen_string_literal: true
 
-class ApplicationDecorator < Draper::Decorator
-  # Define methods for all decorated objects.
-  # Helpers are accessed through `helpers` (aka `h`). For example:
+require 'forwardable'
+
+class ApplicationDecorator
+  extend Forwardable
+
+  # Forwards (and optionally renames) methods to the wrapped object.
+  #
+  # @param methods [Array<Symbol>] the method(s) to forward
+  # @param aliases [Hash<Symbol, Symbol>] the methods to alias
+  # @return [void]
+  # @example Forward some methods
+  #   forward :first_name, :last_name
+  # @example Forward and rename a method
+  #   forward age: :user_age
+  # @example Forward and rename at the same time
+  #   forward forward :first_name, :last_name, age: :user_age
```

Comment:
There could be just one `forward` and still works the same. Guess you double type the `forward`

Reply:
👍 I messed up the documentation there.